### ISSUE TITLE
Autophony

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10143]
**Preferred term label:**
Autophony

**Synonyms**

**Definition (free text, please give PubMed ID)**
The perception that your voice is too loud or echoing in your ears
https://uts.nlm.nih.gov/uts/umls/concept/C0877395

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Eustachian tube dysfunction  https://www.ncbi.nlm.nih.gov/books/NBK555908/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000032
- label: Autophony
- parents: [Hearing abnormality (HP:0000364)](https://hpo.jax.org/app/browse/term/HP:0000364)
- pmid: PMID:34993986|PMID:31750425
- def: Hearing one's own voice resonating loudly in the ear.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10143
- synonyms:
